ha/setup/post_disruptive_upgrade.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# abcheck_for_any_resource_presenceout this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+"""Module to do post-upgrade routines in scope of Disruptive Upgrade feature."""
+
+from cortx.utils.log import Log
+from ha.execute import SimpleCommand
+from ha.const import (BACKUP_DEST_DIR_CONF, CONFIG_DIR)
+from ha.setup.create_pacemaker_resources import create_all_resources
+
+
+def check_for_any_resource_presence() -> None:
+    '''Check if any resources are already present in a cluster.
+       if yes, means, pre-upgrade steps failes hence exit.'''
+
+    Log.info('Check for any resource presence in a cluster')
+
+    list_pcs_resources = 'crm_resource --list-raw'
+    resource_list = SimpleCommand().run_cmd(list_pcs_resources)
+
+    resource_list.split('\n')
+
+    if resource_list:
+         Log.error('Some resources are already present. \
+                    Perform Post-Upgrade again')
+        return
+
+def is_cluster_standby_on() -> None:
+    '''Check if cluster is in standby mode. If not, make standby mode ON'''
+
+    Log.info('Check cluster is in standby mode')
+    check_standby_mode = 'crm_standby --query | awk \'{print $3}\' | cut -d \'=\' -f 2'
+    value = SimpleCommand().run_cmd(check_standby_mode)
+
+    if value.lower() != 'on':
+        Log.error('cluster is not in standby mode. Making standby mode ON')
+        standby_cmd = "pcs node standby --all --wait=600"
```

Comment:
Already we have a "PCS_NODE_STANDBY" command defined in const.py file, please use that.

Reply:
Already we have a "BASE_WAIT_TIME" defined in const.py file for the sleep purpose, please check how it is used in other standby cases and apply it here.

Reply:
Ok. Done.